pkg/routeimporter/config.go

```diff
@@ -5,27 +5,27 @@ import (
 	"github.com/blakewilliams/viewproxy/pkg/fragment"
 )
 
+type ConfigFragment struct {
+	Path             string
+	Metadata         map[string]string
+	IgnoreValidation bool
+	Children         map[string]ConfigFragment
+}
+
 type ConfigRouteEntry struct {
-	Url               string               `json:"url"`
-	LayoutTemplate    *fragment.Definition `json:"layout"`
-	FragmentTemplates fragment.Collection  `json:"fragments"`
-	Metadata          map[string]string    `json:"metadata"`
-	IgnoreValidation  bool
+	Url              string            `json:"url"`
```

Comment:
```suggestion
	Path              string            `json:"url"`
```

Anyone have thoughts on updating this while we're already making breaking changes?

Reply:
This change makes sense to me.